infra/azure/README.md

```diff
@@ -32,6 +32,13 @@ Once terraform has completed successfully, note the `gateway_ip` in the
 output and create an A record for the domain of your choosing for that
 IP with a DNS provider.
 
+There are some resources which, unfortunately, cannot be configured directly
+with terraform. You must now configure those resources by running:
```

Comment:
I believe, at least for the admin consent, that this only needs to be run after the first terraform application (when the SP is created). Are you intending for this to be a place where any post-terraform commands need to go or just on initial setup? I feel like I would want to know that detail if I were maintaining the system.